### PR TITLE
Release 1.0.7 notes

### PR DESCRIPTION
**Pull request by bassemmagdy**
- Adding release notes for version 1.0.7

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
...
+- Enhanced signer selector UI/UX [#1688](https://github.com/symbol/desktop-wallet/issues/1688)
+
+#### Fixed
+- Fixed Aggregate bonded - Failure_Core_Future_Deadline issue [#1490](https://github.com/symbol/desktop-wallet/issues/1490)
+- Fixed multisig account owned mosaic actions visibility [#1679](https://github.com/symbol/desktop-wallet/issues/1679)
+- Fixed Ledger harvesting VRf and Remote keys not being saved [#1692](https://github.com/symbol/desktop-wallet/issues/1692)
+- Fixed pagination in transaction history [#1652](https://github.com/symbol/desktop-wallet/issues/1652)
+- Fixed multilevel multisig behavior when account is a cosigner of multiple trees [#1687](https://github.com/symbol/desktop-wallet/issues/1687)
+- Fixed partial transactions aren't shown correctly [#1697](https://github.com/symbol/desktop-wallet/issues/1697)
+- Fixed mosaic/namespace metadata listing on signer change [#1668](https://github.com/symbol/desktop-wallet/issues/1668)[#1669](https://github.com/symbol/desktop-wallet/issues/1669)[#1670](https://github.com/symbol/desktop-wallet/issues/1670)
```
Comment (AnthonyLaw) [inline CHANGELOG.md:26]: missing new line and `-` on `[#1669]`

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
...
+- Fixed infinite rest calls when transaction sent to expired namespace alias [1695](https://github.com/symbol/desktop-wallet/issues/1695)
+- Reload mosaics only when signer is changed in transaction form [#1684](https://github.com/symbol/desktop-wallet/issues/1684)
+- Fixed form is resetting when a transaction is confirmed [#1683](https://github.com/symbol/desktop-wallet/issues/1684)
```
Comment (AnthonyLaw) [inline CHANGELOG.md:29]: #1683 is not match the issue url.

File: CHANGELOG.md (hunk 1 above)
Comment (bassemmagdy) [reply CHANGELOG.md:26]: The three of them are related and got merged as a one fix.

File: CHANGELOG.md (hunk 2 above)
Comment (bassemmagdy) [reply CHANGELOG.md]: fixed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
...
+- Added accessible terms and privacy policy links[#814](https://github.com/symbol/desktop-wallet/issues/814) 
+
+#### Changed
+- Disable Ledger profile menu item in browser version [#1590](https://github.com/symbol/desktop-wallet/issues/1590)
+- Enhanced signer selector UI/UX [#1688](https://github.com/symbol/desktop-wallet/issues/1688)
+
+#### Fixed
+- Fixed Aggregate bonded - Failure_Core_Future_Deadline issue [#1490](https://github.com/symbol/desktop-wallet/issues/1490)
+- Fixed multisig account owned mosaic actions visibility [#1679](https://github.com/symbol/desktop-wallet/issues/1679)
+- Fixed Ledger harvesting VRf and Remote keys not being saved [#1692](https://github.com/symbol/desktop-wallet/issues/1692)
```
Comment (segfaultxavi) [inline CHANGELOG.md:22]:
```suggestion
- Fixed Ledger harvesting VRF and Remote keys not being saved [#1692](https://github.com/symbol/desktop-wallet/issues/1692)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
...
+- Fixed pagination in transaction history [#1652](https://github.com/symbol/desktop-wallet/issues/1652)
+- Fixed multilevel multisig behavior when account is a cosigner of multiple trees [#1687](https://github.com/symbol/desktop-wallet/issues/1687)
+- Fixed partial transactions aren't shown correctly [#1697](https://github.com/symbol/desktop-wallet/issues/1697)
+- Fixed mosaic/namespace metadata listing on signer change [#1668](https://github.com/symbol/desktop-wallet/issues/1668)[#1669](https://github.com/symbol/desktop-wallet/issues/1669)[#1670](https://github.com/symbol/desktop-wallet/issues/1670)
+- Fixed infinite rest calls when transaction sent to expired namespace alias [1695](https://github.com/symbol/desktop-wallet/issues/1695)
```
Comment (segfaultxavi) [inline CHANGELOG.md:27]:
```suggestion
- Fixed infinite REST calls when a transaction is sent to an expired namespace alias [1695](https://github.com/symbol/desktop-wallet/issues/1695)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
...
+- Reload mosaics only when signer is changed in transaction form [#1684](https://github.com/symbol/desktop-wallet/issues/1684)
+- Fixed form is resetting when a transaction is confirmed [#1683](https://github.com/symbol/desktop-wallet/issues/1683)
```
Comment (segfaultxavi) [inline CHANGELOG.md:29]:
```suggestion
- Fixed form resetting when a transaction is confirmed [#1683](https://github.com/symbol/desktop-wallet/issues/1683)
```

**Review by segfaultxavi — CHANGES_REQUESTED** [review]
Sorry, I had to 😁

**Comment by yilmazbahadir** [comment]
Rebased the branch on top of `dev` and updated the `CHANGELOG.md` with the latest fix.
> Fixed aggregate bonded announced by third party address issue [#1719](https://github.com/symbol/desktop-wallet/issues/1719)

**Review by rg911 — APPROVED** [review]

**Pull request merged**